doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -764,27 +654,13 @@ the toplevel, and using them in source files is discouraged.
    loadpath (see the command `Declare ML Module`` in Section :ref:`compiled-files`).
 
 
-.. cmd:: Print ML Path @string
+.. cmd:: Print ML Path
 
    This command displays the current OCaml loadpath. This
    command makes sense only under the bytecode version of ``coqtop``, i.e.
    using option ``-byte``
```

Comment:
It's deprecated and a no-op in `coqargs.ml` but still seems to do something in coqProjec_file.ml:

`  | ("-no-opt"|"-byte") :: r -> aux { proj with use_ocamlopt =  false } r`

Remove it?

Reply:
This file is depended upon by both `coq_makefile` and CoqIDE.

Running `coqide -byte` leads to the failure:
```
Coqtop exited
Command was .../bin/coqidetop.opt -nois -batch -byte
Answer was: Unknown option -byte
```

So either CoqIDE should be fixed so that the `-byte` option makes it call `../bin/coqidetop.byte` (although this is not likely to be installed on the users' machines) or the option should be removed (simpler).
For `coq_makefile`, the option seems to be handled, but it would have to be tested (and documented).

Reply:
You want to submit an issue?

Reply:
No, in fact I misunderstand the issue: I was wrong to consider that `-byte` passed to `coqide` has any relation to this code. `coqide -foo` behaves the same and fails similarly. In fact, any option that `coqide` does not know is passed directly to `coqidetop`.

We could open just a documentation issue saying that the `_CoqProject` format apparently supports the `-no-opt`  `-byte` options but these are not documented.

Reply:
See #12062.